src/pages/rental-details/rental-details.ts

```diff
@@ -21,6 +21,12 @@ export class RentalDetailsPage {
 
   ngOnInit() {
     this.items = this.navParams.get('items');
+
+    const today = new Date();
+    let tomorrow = new Date();
```

Comment:
Interesting, I thought `const` was the equivalent of a read only variable, but it simply means that it won't be reassigned anywhere else. So object properties are mutable.